README.md

```diff
@@ -78,6 +78,51 @@ createAction('ADD_TODO')('Use Redux');
 
 `metaCreator` is an optional function that creates metadata for the payload. It receives the same arguments as the payload creator, but its result becomes the meta field of the resulting action. If `metaCreator` is undefined or not a function, the meta field is omitted.
 
+##### `createActions(?actionsMap, ?...actionTypes)`
+
+Returns an object mapping action types to action creators. The keys of this object are camel-cased from the keys in `actionsMap`; the values are the action creators.
+
+`actionsMap` is an optional object with action types as keys, and whose values can be
+
+- a function, which is the payload creator for that action
+- an object with `payload` and `meta` keys, mapping to payload and meta creators
+    - `meta` is **required** in this case to distinguish it meaningfully from the function form above
```

Comment:
Ah, I understand now. It was indeed a bit confusing.

The only problem I now see is, it is not really self-explaining if you see the function form and the object form together for the first time, but they are basically the same, except that the object form has additionally `meta`.

How about inheriting the function form from `createAction` and write `actionMap` like this:

```js
ACTION_ONE: [
  (first, second) => [first, second],
  (first, second) => {first, second]
]
```

The first value in the array is `payload` and second `meta`.

Just an idea to reduce the possibilities how we can write the `actionMap`.